src/endpoints/substrate-endpoint/services/service-request.service.ts

```diff
@@ -55,6 +55,10 @@ export class ServiceRequestService {
         [country: string]: RequestByCountryDictInterface;
       } = {};
       for (const req of hits) {
+        if (!req?._source?.service_request) {
+          continue;
+        }
```

Comment:
Is this a `null` value check? It's better to use it like `req?._source?.service_request !== null`.